### DISCUSSION TITLE
Scheduler Info

### DISCUSSION THREAD

**Discussion by HeyItsJono** [d4rken-org/sdmaid-se#1367] [Q&A]
Hi, thanks for the app! Just wanted to check; does the scheduler auto-delete as well, or just run the search for the relevant tasks and allow the user to manually review & delete later?

**Comment by MarkyMarkDE** [comment]
No, there is no option for and it is little out of the sense of the app 😉

**Reply by d4rken** [reply]
Also:

> manually review & delete later

This would not work well as results are only stored in memory. If there is no active task, then SD Maid loses priority and the system is likely to kill the process, so when you would open it later, it wouldn't show the results anymore.

**Reply by MarkyMarkDE** [reply]
aahhh okay, thanks for the advanced background information about that.

**Reply by HeyItsJono** [reply]
@MarkyMarkDE Sorry, your answer is a little confusing; is the "no" saying that it doesn't auto-delete, or is the "no" saying that you can't manually review the results and delete at a later date?

@d4rken If that's the case then as it stands the app may automatically delete files the user has no intention of deleting, which removes the ability for those of us who have paid for Pro to use the Scheduling feature if we aren't up for the risk of randomly losing an important file. Would it not be possible to store a list of results in a tempfile until user review?

**Reply by MarkyMarkDE** [reply]
@HeyItsJono My "no" means, the second one. You can't search for trash in Cache and delete it later, this is little against the sense of the App and darken has explained the technical background.
